Formula/star.rb

```diff
@@ -15,12 +15,13 @@ class Star < Formula
     sha256 "410f5637ccdf115373b5a08c5037cdb8c66cb113719ead191070d087eae43285" => :mavericks
   end
 
+  depends_on "gcc" => :build
```

Comment:
does it no longer work with llvm?

Reply:
tried both llvm and gcc, unfortunately, same build issue (could not find some file).

I think it could be built with llvm (revert my commits to avoid confusion)